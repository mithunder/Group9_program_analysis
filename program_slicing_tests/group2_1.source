module group21 :
	x := 5;
	y := 1;
	y := 3;
	x := 5;
	do x > 1 -> {
		y := x * y ;
		@programslicing="true"
		x := x - 1
		}	
	od;	
	z := x
end
