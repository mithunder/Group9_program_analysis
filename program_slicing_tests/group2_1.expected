module group21 :
x := 5;
do
   (x>1) ->
      x := (x-1)
od
end
